.github/workflows/ci.yml

```diff
@@ -180,13 +180,21 @@ jobs:
       working-directory: ${{github.workspace}}/install
       run: ./f3d.app/Contents/MacOS/f3d ../source/testing/data/suzanne.obj --output=../output.png
 
+    # TODO: This job should be replaced by a useful example using the libf3d API and merged with the next job
```

Comment:
good point, do you want to do it in this MR ?

Reply:
Probably later, we have to think about useful examples that showcase the power of libf3d, not just something to test that the SDK is working.